Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4865367)